libs/sdk-ui-kit/src/Datepicker/Datepicker.tsx

```diff
@@ -93,6 +99,14 @@ function parseDate(str: string, dateFormat: string): Date | undefined {
     }
 }
 
+function convertWeekStart(weekStart: WeekStartType): 0 | 1 | 2 | 3 | 4 | 5 | 6 {
```

Comment:
Possibly, you can write the type as `DayPickerProps["weekStartsOn"]` to not repeat this weird inner type of DayPicker component.